contribution.json

```diff
@@ -12,5 +12,11 @@
     "linkedIn": "https://www.linkedin.com/in/rishabh-chaudhari-015526225/",
     "project-name": "WEb Dev",
     "project-link": "https://github.com/Raporaz03/webdev.github.io"
+  },
+  {
+    "name": "Anuragh Barua",
+    "github": "https://github.com/anuragh2002",
+    "linkedIn": "https://www.linkedin.com/in/anuragh-barua-796b05135/",
+    "twitter": "https://twitter.com/AnuraghBarua/"
```

Comment:
add "project-name": "added links to opinco website" or something like this and "project-link": link to your pr that you created don't leave any fields